src/ekat/util/ekat_test_utils.cpp

```diff
@@ -1,13 +1,18 @@
 #include <ekat/util/ekat_test_utils.hpp>
-#include <ekat/ekat_assert.hpp>
+#include "ekat/ekat_assert.hpp"
```

Comment:
Any reason for not changing the line 1 to use quotes, too? Also: are you sure you want to use angle braces for Kokkos_Core.hpp?

Reply:
You're right, ekat's headers should use quotes rather than braces. For kokkos, I am leaning on braces, since, even if we build it as part of the project, it semantically is a TPL. We don't develop it, and we don't test it. We only build it on the fly cause it's cheap, and avoids the headache of having it pre-installed (and maintained) on all machines.